src/main/resources/db/migration/V1__init_db.sql

```diff
@@ -133,6 +133,10 @@ CREATE TABLE POSTS
 ALTER TABLE POSTS
     OWNER TO POSTGRES;
 
+ALTER TABLE POSTS
+    ADD COLUMN VIDEO_URL VARCHAR,
```

Comment:
@melnikovae87 Should we put default value "null" for this column? What's ur opinion?

Reply:
If this field is optional for different typo of posts